src/Data.php

```diff
@@ -24,19 +24,19 @@ class Data
     public $mappingCleanNameMethodGroup = array();
 
     // 1st array used, stores the csv of country to zone
-    private $mappingCountryToZone = array();
+    public $mappingCountryToZone = array();
```

Comment:
I wouldn't recommend having any public class variables. Have getters and setters if someone _really_ needs to change them but ideally you want to minimise the ways in which state can be changed.